bot.py

```diff
@@ -747,7 +759,7 @@ async def rewards(ctx, *args):
             msg = "Available rewards```\n"
         # Get prizes for a specific category
         elif len(args) == 1:
-            rewards_dict = firebase.get_rewards(args[0])
+            rewards_dict = firebase.get_rewards(args[0].lower())
```

Comment:
I would suggest making this lower() command for any places where there are arguments (people's names, prizes, etc.)